.github/workflows/dev-cicd.yml

```
@@ -63,4 +63,18 @@ jobs:
               -p 8080:8080 \
+              -e SPRING_PROFILES_ACTIVE=dev \
               ${{ secrets.DOCKER_USERNAME }}/onmoim:develop
+
+      - name: Send Discord Notification on Failure
+        if: failure()
+        uses: sarisia/actions-status-discord@v1
+        with:
+          webhook: ${{ secrets.DISCORD_WEBHOOK_URL }}
+          status: ${{ job.status }}
+          title: "CI/CD 빌드 실패"
+          description: |
```

actionlint (1.7.10)

property "id" is not defined in object type {check_run_id: number; container: {id: string; network: string}; services: {string => {id: string; network: string; ports: {string => string}}}; status: string}

(expression)